.github/workflows/extension_ci.yml

```diff
@@ -39,72 +39,40 @@ jobs:
           prefix-key: "pgmq-extension-lint"
           workspaces: |
             pgmq/
-          # Additional directories to cache
-          cache-directories: |
-            /home/runner/.pgrx
-      - uses: ./.github/actions/pgx-init
-        with:
-          working-directory: ./
       - name: Cargo format
-        run: cargo +nightly fmt --all --check
+        run: cargo +nightly fmt --manifest-path integration_test/Cargo.toml --all --check
       - name: Clippy
-        run: cargo clippy
+        run: cargo clippy --manifest-path integration_test/Cargo.toml
 
   test:
     name: Run tests
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        pg: [
-          {version: "12.19", pgrx_version: "pg12"},
-          {version: "13.15", pgrx_version: "pg13"},
-          {version: "14.12", pgrx_version: "pg14"},
-          {version: "15.7", pgrx_version: "pg15"},
-          {version: "16.3", pgrx_version: "pg16"}
-        ]
+        pg: [12, 13, 14, 15, 16]
+    container: pgxn/pgxn-tools
     steps:
+      - run: pg-start ${{ matrix.pg }}
       - uses: actions/checkout@v2
+      - run: apt install -y pkg-config
```

Comment:
Hrm, what's this for? Maybe I should add it to pgxn-tools?

Reply:
Our tests are still written in rust, and needed this to compile.